source/client/controllers/signup-controller.js

```diff
@@ -0,0 +1,53 @@
+Space.messaging.Controller.extend(Space.accountsUi, 'SignupController', {
```

Comment:
Do we want to keep using `Space.messaging.Controller` as a base? I think it's a bit generic and was hoping we would move away to more specific controller types using the mixins. This is a perfect example as a SignupController is just a client-side EventSubscriber and EventPublisher.